performance_regression_test.py

```diff
@@ -227,19 +229,32 @@ def run_mixed_workload(self, nemesis=False):
     def run_workload(self, stress_cmd, nemesis=False):
         # create new document in ES with doc_id = test_id + timestamp
         # allow to correctly save results for future compare
+        latency_results_file = 'latency_results.json'
         sub_type = 'read' if ' read ' in stress_cmd else 'write' if ' write ' in stress_cmd else 'mixed'
+        if os.path.exists(latency_results_file):
+            os.remove(latency_results_file)
         self.create_test_stats(sub_type=sub_type, doc_id_with_timestamp=True)
+
```

Comment:
i think, it is better to move out to new method code, where you remove file and may be move name of file configurable

under which directory the file will be stored?

Reply:
```
def  get_latency_result_file():
    latency_results_file = 'latency_results.json'
    if os.path.exists(latency_results_file):
        os.remove(latency_results_file)
     return latency_results_file
latency_results_file = get_latency_results_file() 
```